cypress/e2e/CMS/$CreateAnImage.feature

```diff
@@ -0,0 +1,152 @@
+Feature: User creating an image
+
+Scenario: create a new image
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on "Media" sub tab
+        And user clicks on "Add media" action button
+        And user clicks on Image content type
```

Comment:
Image should not be hardcoded - this step exists in common as 
And('user clicks on {string} content type', (contentType) => {

Reply:
done